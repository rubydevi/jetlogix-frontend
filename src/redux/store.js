import { configureStore } from '@reduxjs/toolkit';
import aeroplanesReducer from './aeroplanes/aeroplanesSlice';
import reservationsReducer from './reservations/reservationSlice';

const store = configureStore({
  reducer: {
    aeroplanes: aeroplanesReducer,
    reservation: reservationsReducer,
  },
});

export default store;
