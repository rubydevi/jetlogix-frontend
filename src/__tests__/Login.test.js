import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/store';

import Login from '../components/Login';

test('renders Login component correctly', () => {
  const { container } = render(
    <Provider store={store}>
      <BrowserRouter>
        <Login />
      </BrowserRouter>
    </Provider>,
  );
  expect(container).toMatchSnapshot();
});
