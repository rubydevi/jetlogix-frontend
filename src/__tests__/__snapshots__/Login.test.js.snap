// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Login component correctly 1`] = `
<div>
  <section
    class=" rounded-1 login-register-style"
  >
    <p
      aria-live="assertive"
      class="offscreen"
    />
    <h1
      class="header"
    >
      Sign In
    </h1>
    <form
      class=""
    >
      <label
        for="email"
      >
        Email:
      </label>
      <input
        autocomplete="on"
        id="email"
        required=""
        type="email"
        value=""
      />
      <label
        for="password"
      >
        Password:
      </label>
      <input
        autocomplete="on"
        id="password"
        required=""
        type="password"
        value=""
      />
      <button
        class="btn-color text-light"
        type="submit"
      >
        Sign In
      </button>
    </form>
    <p>
      Need an Account?
      <br />
      <a
        class="text-light"
        href="/register"
      >
        <span
          class="text-login"
        >
          Sign Up
        </span>
      </a>
    </p>
  </section>
</div>
`;
